crates/sui-indexer-alt/src/db/tempdb.rs

```diff
@@ -0,0 +1,345 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Is this file duplicated? Lets not duplicate code if possible. Instead we should go ahead and split the tempdb logic into its own crate so that it can be reused for better maintainability